Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         python3 \
         runit \
         gnupg-agent \
-	socat \
+        socat \
```

Comment:
`socat` change is just adding additional spaces to match the tabbing on other elements in this block.